documentation/cloud-manager-guide/environments/environment-details/environment-overview.md

```diff
@@ -0,0 +1,145 @@
+---
+keywords:
+title: Environment Overview
+description: Components and navigation for an environment's detailed view
+---
+
+# Environment Overview
+
+Each environment has a detailed view where you can see further information about the environment, monitor its status, and perform various operations on the environment. This guide outlines the features of the *Overview* screen and how to navigate the detailed view of an environment.
+
+## Getting started
+
+To navigate to the detailed view of an environment, select the environment name from the list of environments displayed on the *Environments* screen.
+
+![image description](images/overview-select-env.png)
+
+This brings you to the environment's *Overview* screen that displays a summary of the environment details and operation statuses. Further monitoring and updating tools for the environment are accessible from the navigation bar located at the top of the page.
+
+![image description](images/overview-screen.png)
+
+## Top navigation
+
+A navigation bar is located at the top of the *Overview* screen and is visible from all tabs in the environment details view. The top navigation allows you to access several monitoring and updating tools through the following tabs:
+
+- Overview
+- Logs 
+- Backups
+- Configuration
+- Activity Log
+
+![image description](images/overview-topnav.png)
+
+### Logs
+
+The *Log* screen allows you to view an environment's log details, imported from Elastic.
```

Comment:
Should be "tab" instead of "screen" to be consistent.

Reply:
Yes, users should be familiar with Elastic (https://www.elastic.co/)